README.md

```diff
@@ -14,17 +14,19 @@ Steps:
   - [1.2 Installation](#12-installation)
     - [SCT Installation](#sct-installation)
     - [dcm2niix Installation](#dcm2niix-installation)
+    - [FSLeyes Installation](#fsleyes-installation)
     - [Downloading this repository](#downloading-this-repository)
 - [2. Data structure](#2-data-structure)
   - [2.1 File organization](#21-file-organization)
-  - [2.2 DICOM to NIfTI conversion](#22-dicom-to-nifti-conversion)
+- [3. Analysis pipeline](#3-analysis-pipeline)
 
 ## 1. Getting Started
 
 ### 1.1 Dependencies
 
 * [Spinal Cord Toolbox v6.4](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.4): toolbox for processing spinal cord MRI data
 * [dcm2niix >= v1.0.20220505](https://github.com/rordenlab/dcm2niix?tab=readme-ov-file#install): tool for converting DICOM images into the NIfTI format
+* [FSLeyes](https://owncloud.cesnet.cz/index.php/s/z5h02r0cq0B7ESf): tool for visualizing NIfTI images
 
 ### 1.2 Installation
```

Comment:
`wget https://github.com/spinalcordtoolbox/spinalcordtoolbox/archive/refs/tags/6.4.zip`

wget is not already installed, so we need to install hombrew and `brew install wget`

Reply:
for me git is already installed

Reply:
maybe try `curl`

Reply:
Can you please try

```bash
curl -L -o 6.4.zip https://github.com/spinalcordtoolbox/spinalcordtoolbox/archive/refs/tags/6.4.zip
```

Reply:
it works!

Reply:
Nice! I changed `wget` to `curl` in https://github.com/sct-pipeline/balgrist-sci/pull/5/commits/500a7e06190e3cd204b4e0f72114af2807ce1cde